app/src/App.tsx

```diff
@@ -40,9 +41,11 @@ const App = () => {
 
   return (
     <Provider store={defaultStore}>
-      <NavigationContainer ref={navigationRef}>
-        <StackNavigator />
-      </NavigationContainer>
+      <SafeAreaProvider>
```

Comment:
It seems like this package will change UI spacing for every page. Did you check to make sure every page has the same, or better UI as before?

Reply:
Yeah, I manually tested on iOS and Android, so notched and notchless. Looks good to me.

Reply:
Ah, actually there's a tiny tiny amount of movement on iOS when the dialer is set to client vs. number. Let me see if I can fix it.